### ISSUE TITLE
Concept NavItem disabled state

### ISSUE THREAD

**Issue by mj-hof** [SchwarzIT/onyx#1862]
### Discussed in https://github.com/SchwarzIT/onyx/discussions/1837

- [ ] Check if an additional info, like tooltip, is needed to give user enough info why item is disabled

**Comment by flubnau** [comment]
As if a tooltip does not work on keyboard Navigation, because an disabled button could not be focused it won't work on that use case. Same for Touch usage.

_But if you use only the keyboard, there’s no way of seeing that tooltip because the button cannot be focused with disabled. The same thing happens in touch devices too. Ouch!_

**Comment by Sinan-SchwarzIT** [comment]
Hi @mj-hof, is this feature implemented or planned? I haven't seen a disabled property so far :-)

**Comment by larsrickert** [comment]
@Sinan-SchwarzIT The disabled state is concepted from UX in Figma but we have to implement it technically in #2238.
I marked the ticket for refinement so we will talk about it in our next refinement meeting :)
